### PR TITLE
dbcat support to 3.10

### PR DESCRIPTION
**Pull request by yjagdale**
added support for m1 and python 3.10

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -23,11 +23,11 @@ repository="https://github.com/tokern/dbcat/"
 
 
 [tool.poetry.dependencies]
-python = ">=3.6.1,<3.9"
+python = ">=3.7,<=3.10.8"
```
Comment (vrajat) [inline pyproject.toml:26]: Should this be `>=3.8` since the tests run only for 3.8 onwards?

Comment (yjagdale) [reply pyproject.toml:26]: 👍

File: pyproject.toml [hunk 2]
```diff
@@ -43,7 +43,8 @@ dataclasses = {version = ">=0.6", markers="python_version >= '3.6' and python_ve
 acryl-datahub = {version = "^0.8.16", optional = true}
 great-expectations = {version = "^0.13.42", optional = true}
 pydantic = "^1.8.2"
-SQLAlchemy-Utils = "^0.37.9"
+greenlet = "^2.0.1"
```
[the remark this answers is not in the document]
Comment (yjagdale) [reply pyproject.toml:46]: for Python version 10, greenlet is not being pulled and is expected to be available explicitly(test cases failed with the error module greenlet not found).

[the remark this answers is not in the document]
Comment (yjagdale) [reply pyproject.toml]:
```_____________________ ERROR collecting test/test_export.py _____________________
ImportError while importing test module '/home/runner/work/dbcat/dbcat/test/test_export.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test/test_export.py:7: in <module>
    from dbcat.datahub import CatalogSource
dbcat/datahub.py:12: in <module>
    from datahub.ingestion.source.sql.sql_common import (
../../../.cache/pypoetry/virtualenvs/dbcat-Q-0cZex8-py3.8/lib/python3.8/site-packages/datahub/ingestion/source/sql/sql_common.py:98: in <module>
    from datahub.utilities.sqlalchemy_query_combiner import SQLAlchemyQueryCombinerReport
../../../.cache/pypoetry/virtualenvs/dbcat-Q-0cZex8-py3.8/lib/python3.8/site-packages/datahub/utilities/sqlalchemy_query_combiner.py:12: in <module>
    import greenlet
E   ModuleNotFoundError: No module named 'greenlet'```

**Review by vrajat — APPROVED** [review]

**Pull request merged**